### PR TITLE
Frontend issues

### PR DESCRIPTION
**Pull request by tulvit**
### Maintaining HTML code
- Deleting excessive line breaks.
- Extra formatting for better readability.
- Improving overall consistency among code.
- Fixing w3.org validation errors (like moving script tags inside body tag).
### Responsiveness 
- Adding meta view port tag.
- Fixing bugs for mobiles.
- Fixing bugs for screens in range 992px-1200px (one of the standard Bootstrap breakpoint).
- Fixing notify.js tooltip for mobiles (basically, re-positioning it for <992px screens).
### Misc
- Updating jQuery version from v2 to v3 (works fine with no revealed issues).
- Commenting out "Preset" and "Additional options" sections (no need in it now).

### CODE REVIEW

File: webui/app/static/js/index.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ $(document).ready(function () {
             event.preventDefault();
             // show notification and exit handler
             return $(".file-input").notify("Please, select file for uploading", {
-                position: "right",
+                position: ($(window).width() < 992) ? "bottom right" : "right",
```
Comment (joeflack4) [inline webui/app/static/js/index.js]: Works good! Will follow up with you separately on doing the same with other notifications (i.e. conversion errors).
please_select_iphone-6-side-orientation
errors_iphone-6-side-orientation

File: webui/app/templates/index.html [hunk 2]
```diff
@@ -41,6 +41,7 @@ <h2 class="text-center">Conversion Tool Web UI</h2>
           <input type="text" value="English" name="language" class="w100prc txt-input"/>
         </div>
       </div>
+      <!--
```
Comment (joeflack4) [inline webui/app/templates/index.html]: Perfect, thanks. I'll let you know when we intend to add these back.

File: webui/app/templates/index.html [hunk 3]
```diff
@@ -1,117 +1,117 @@
...
+          </label>
+        </div>
+      </div>
+      <div class="col-md-4 col-md-offset-4">
+        <p>Language:</p>
+        <div class="form-group">
+          <input type="text" value="English" name="language" class="w100prc txt-input"/>
+        </div>
+      </div>
+      <!--
```
Comment (joeflack4) [inline webui/app/templates/index.html]: Perfect, I'll let you know when to add these back.

File: webui/app/static/css/styles.css [hunk 4]
```diff
@@ -4,4 +4,24 @@
 
 .txt-input {
     padding: 3px 7px;
-}
\ No newline at end of file
+}
+
+@media (max-width: 370px),
```
Comment (joeflack4) [inline webui/app/static/css/styles.css]: I am trying to remember. Could you send me a screenshot of what these `@media` properties are doing? Or do they only apply if we uncomment out the options that we have hidden for now?

File: webui/app/templates/index.html [hunk 5]
```diff
@@ -1,117 +1,117 @@
...
-    <meta charset="UTF-8">
-    <link href="{{ url_for('static', filename='css/bootstrap.min.css') }}"
-          rel="stylesheet" type="text/css"/>
-    <link href="{{ url_for('static', filename='css/fileinput.min.css') }}"
-          rel="stylesheet" type="text/css"/>
-    <link href="{{ url_for('static', filename='css/styles.css') }}"
-          rel="stylesheet" type="text/css"/>
-    <title>Conversion UI</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (joeflack4) [inline webui/app/templates/index.html]: Thanks for spotting and solving this issue!

File: webui/app/templates/index.html [hunk 6]
```diff
@@ -120,15 +120,10 @@ <h2 class="text-center">Conversion Tool Web UI</h2>
...
-        type="text/javascript"></script>
-<script src="{{ url_for('static', filename='js/libs/fileinput.min.js') }}"
-        type="text/javascript"></script>
-<script src="{{ url_for('static', filename='js/libs/bootstrap.min.js') }}"
-        type="text/javascript"></script>
-<script src="{{ url_for('static', filename='js/libs/notify.min.js') }}"
-        type="text/javascript"></script>
-<script src="{{ url_for('static', filename='js/index.js') }}"
-        type="text/javascript"></script>
-</html>
```
Comment (joeflack4) [inline webui/app/templates/index.html]: For this one, where you have multiple small commits doing the same thing, James introduced me to `git commit --amend`, which will add your latest updates to the most recent commit and allow you to use the same message / write a new message.

For more complicated squashing of commits, we use `git rebase -i HEAD~#` (replacing "#" with 5 will allow you to combine the last 5 commits, and replacing both "HEAD" and "#" should let you choose any commit range).

screen shot 2017-12-12 at 11 27 14 am

**Comment by joeflack4** [comment]
Good work, Vitaly! Merging this. Could you follow up on follow-up topics (1) and (2)?

**Comment by tulvit** [comment]
@joeflack4 Thanks!

Topic 1: Done! Will be committed soon!
Topic 2: I've sent you some explanations.

**Pull request merged**